src/components/App.tsx

```diff
@@ -1,12 +1,46 @@
 import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
fix conflict. You should not disable eslint here

Reply:
done